src/style.css

```
@@ -1 +1,114 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  font-style: normal;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 600;
+  font-style: normal;
+  src: url("./fonts/Avenir-Heavy.ttf");
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  padding: 0;
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+}
+
+.form {
+  display: flex;
+  margin-top: 20px;
+  justify-content: center;
+}
+
+.form-big {
+  width: 100%;
+  height: 70px;
+}
+
+.field {
+  position: relative;
+  display: block;
+  width: 100%;
+}
+
+.search {
+  width: 100%;
+  height: 100%;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  color: #3d4e61;
+  font-weight: 300;
+  font-family: inherit;
+  box-shadow: 0 1px 8px rgb(61 78 97 / 10%);
+  outline: none;
+}
+
+.searchBig {
+  font-size: 16px;
+  padding-left: 62px;
+  line-height: 22px;
+}
```

```suggestion
.search {
  width: 100%;
  height: 100%;
  border: 1px solid #e1e7ed;
  border-radius: 4px;
  color: #3d4e61;
  font-weight: 300;
  font-family: inherit;
  box-shadow: 0 1px 8px rgb(61 78 97 / 10%);
  outline: none;
  font-size: 16px;
  padding-left: 62px;
  line-height: 22px;
}
```

There is no need to create a new class, if you have only two inputs, then one of them can be default, that is, for the larger input in this case we use only the class search. But for the small one we need modifiers that change the properties that distinguish the small input from the big one. That is, for a small input we need an additional class, but not for a large one